src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -115,9 +125,54 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
 
     .addCase(setPosition, (state, { payload: { id, position } }) => {
       state.positionsMap[id] = position;
-    })
-
-    .addCase(setProposal, (state, { payload: { lineAddress, positionId, proposalId } }) => {
+      console.log('Updated Positions Map: ', state.positionsMap[id]);
+    })
+
+    .addCase(makeProposal, (state, { payload: { maker, position } }) => {
```

Comment:
should def be in `addCredit` action. Can make a helper func to abstract all this hardcoded logic if u need to